lib/mayaUsd/ufe/UsdTRSUndoableCommandBase.h

```diff
@@ -80,24 +78,30 @@ class MAYAUSD_CORE_PUBLIC UsdTRSUndoableCommandBase
     virtual void addEmptyAttribute() = 0;
     virtual bool cannotInit() const;
 
+    // Create a UsdSceneItem conditionaly in the first access from the path
+    // EveryTime unde/redo is called, the UsdSceneItem/UsdPrim can go stale.
+    void conditionalCreateItem();
+
 private:
-    inline UsdAttribute attribute() const {
-      return prim().GetAttribute(attributeName());
+    inline UsdAttribute attribute() {
+        return prim().GetAttribute(attributeName());
     }
```

Comment:
@ppt-adsk can't be const since prim is setting a private member variable.

Reply:
Making fItem mutable will allow you to restore const to prim() and to this method as well.